daemon-tests/src/lib.rs

```diff
@@ -280,10 +280,10 @@ pub async fn start_from_open_cfd_state(
     confirm!(lock transaction, order_to_take.id, maker, taker);
     wait_next_state!(order_to_take.id, maker, taker, CfdState::Open);
 
-    (maker, taker, order_to_take.id, fee_structure)
+    (maker, taker, order_to_take.id, predict_fees)
 }
 
-pub struct FeeStructure {
+pub struct PredictFees {
```

Comment:
🙃 What about `FeeCalculator`? Constructing it with `new` is no longer misleading. And calling `calculate_for_hours` on it sounds more natural.